### ISSUE TITLE
babel-node is not recommended for production

### ISSUE THREAD

**Issue by vasco3** [cloverfield-tools/universal-react-boilerplate#48]
I just saw that babeljs docs advice against using babel-node on production like we are doing in https://github.com/cloverfield-tools/universal-react-boilerplate/blob/master/package.json#L7

ref. https://babeljs.io/docs/usage/cli/#babel-node

So should it be compiled to a bin/ dir and point the start script there?

**Comment by ericelliott** [comment]
Yes, we should compile for prod. I typically compile to a `build` directory.

**Comment by vasco3** [comment]
What do you think of running it with PM2 after transpiling with babel?

**Comment by ericelliott** [comment]
That's fine. =)

**Comment by ericelliott** [comment]
I'm open to a PR to document production use, if you'd like to contribute. =)

**Comment by vasco3** [comment]
I'm trying to run the server but don't get anything served. 
```
npm run build
npm run start # or npm run server
```
The app logs in the console that the server is running, but there is nothing served on the browser when I hit `localhost:3000`

Is that still work in progress?

**Comment by ericelliott** [comment]
Probably. There is an outstanding PR request that adds more features [here](https://github.com/cloverfield-tools/universal-react-boilerplate/pull/56). Have you looked at that?

**Comment by ericelliott** [comment]
At any rate, I'd love to accept a PR that displays the parsed README from the markdown for the homepage. =)

**Comment by vasco3** [comment]
Yeah I have seen the outstanding PR. I think I'll debounce my work on this issue until we get the app served to the browser.

**Comment by ericelliott** [comment]
> I'm trying to run the server but don't get anything served.

Interesting. I see "Your App" when I run it. =) Don't you?

**Comment by vasco3** [comment]
ah, for some reason it doesn't work from my vagrant. I tried it in my local and it works thanksf

**Comment by ericelliott** [comment]
Interesting... old version of Node?

**Comment by vasco3** [comment]
it is old.. 0.12.7
but it works if I curl inside vagrant. But not from outside. It only happens with this app. The other apps I can curl them from outside the vagrant box without problems. I'll keep digging into this.

**Comment by ericelliott** [comment]
odd. Sounds like a vagrant thing. =)

**Comment by vasco3** [comment]
found it. It was the `'localhost',` in https://github.com/cloverfield-tools/universal-react-boilerplate/blob/master/source/server/index.js#L18
Since I was using the IP of my vagrant box instead of localhost:3000
eg. `http://192.168.33.10:3000/`

**Comment by ericelliott** [comment]
Want to PR, or is your problem solved? =)

**Comment by vasco3** [comment]
Yeah I can PR. I just wonder why was that `'localhost'` there

**Comment by ericelliott** [comment]
I don't think it's there for any particular reason.

**Comment by nkbt** [comment]
Host fixed by 7b80d37

**Comment by vasco3** [comment]
@nkbt this issue is related to running the app with node instead of babel-node. Nothing to do with the host

**Comment by nkbt** [comment]
I thought we compile for prod, sorry
